src/common/options.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <string>
+#include <cstdlib>
+
+
+namespace shar {
+
+struct Options {
+  std::string url;                           // url to stream to
```

Comment:
Should we initialize all options with default values? (url, codec...)

Reply:
`url` is required option, so it does not need default value
default value for `codec` is empty string, which means that codec should be auto-detected